PaTTAN Room Displays/PaTTAN Room Displays/DualRoomPage.xaml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PaTTAN_Room_Displays.DualRoomPage"
+             Title="DualRoomPage"
+             BackgroundColor="White">
+    <StackLayout>
```

Comment:
I'd suggest a ContentView or ScrollView here rather than a StackLayout

Reply:
I'll look into that tomorrow.  Perhaps we can look at that together?